addons/web/static/src/js/core/dom.js

```diff
@@ -342,15 +343,14 @@ var dom = {
             // a 'real' debounce creation useless. Also, during the debouncing
             // part, the button is disabled without any visual effect.
             $button.addClass('o_debounce_disabled');
+            this.trigger_up('btn_spin_request', {btn: $button[0], spin: true});
```

Comment:
`makeButtonHandler` is a standalone function, it cannot use `this` which is `undefined`. See other comments.

Reply:
:+1: